### PR TITLE
Remove 19h1 and add 21H1 to Helix test queue.

### PR DESCRIPTION
**Pull request by kythant**

### CODE REVIEW

File: build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml [hunk 1]
```diff
@@ -30,8 +30,8 @@ jobs:
 
     # The target queues to run the tests on.
     # Note: %3b is the escape sequence for ';' which is used as the delimiter
-    helixTargetQueuesOpen: 'Windows.10.Amd64.ClientRS5.Open.reunion%3bWindows.10.Amd64.Client19h1.Open.reunion%3bWindows.10.Amd64.Client20h2.Open.reunion'
-    helixTargetQueuesClosed: 'Windows.10.Amd64.ClientRS5.reunion%3bWindows.10.Amd64.Client19h1.reunion%3bWindows.10.Amd64.Client20h2.reunion'
+    helixTargetQueuesOpen: 'Windows.10.Amd64.ClientRS5.Open.reunion%3bWindows.10.Amd64.Client20h2.Open.reunion%3bwindows.10.amd64.client21h1.open.reunion-westus2-v2-rg'
```
Comment (jefgen) [inline build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml:33]: Does the new 21h1 image actually have "westus2-v2-rg" as part of the name?

Comment (kythant) [reply build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml:33]: Oh is that part not needed?

Comment (jefgen) [reply build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml:33]: The other ones don't have it, and I don't think that it is part of the new names.
I think it's just:
`Windows.10.Amd64.Client21H1.Open.reunion`

**Review by jefgen — APPROVED** [review]

**Pull request merged**